### ISSUE TITLE
Enterprise: Share labels within an organization

### ISSUE THREAD

**Issue by begin-again** [isaacs/github#1279]
In my workplace we have many repositories which utilize the same labels. It would be super convenient to be able to create the labels in the Org rather than replicate them for each repository.

**Comment by nombrekeff** [comment]
Yup this would be really nice! I think its related to #1448, an issue I opened.

**Comment by nombrekeff** [comment]
@begin-again It seams the issue I opened is having some attention, check it out and see if it fits into your needs! #1448

**Comment by begin-again** [comment]
@nombrekeff importing/exporting would be a start but then I could expect to be regularly syncing a repository's labels (we have many repositories). Organization level labels would work much better for GH Enterprise where the repositories are not going to be moving between organizations. On a rare occasion that this occurred, GHE could inform the user that they would lose their org labels or potentially convert those org labels to local labels.

**Comment by nombrekeff** [comment]
@begin-again Yup that would be nice, I added a comment referencing to this on the other issue!

**Comment by sypets** [comment]
It appears this is a duplicate or at least similar to #281.

Also, a solution using .github/settings.yml was mentioned with this [comment](https://github.com/isaacs/github/issues/281#issuecomment-381427359):

https://github.com/apps/settings

It still maintains the issues per repo, but should make it easier to unify them across all repositories in the organization via this file.

**Comment by nombrekeff** [comment]
This can now be done _built-in_, check this other comment: https://github.com/isaacs/github/issues/1448#issuecomment-565053400

**Comment by nombrekeff** [comment]
I would suggest closing this issue, if nobody has any objections
